### PR TITLE
Small Updates 

### PR DESCRIPTION
**Pull request by Kayanski**
- Change parse_network return type + exposed supported networks
- Allows possibility to not enable any logs (using env)
- Add the possiblity to set a fee_granter for transactions (Closes ORC-20)
- Sort contracts by id in snapshot testing (for stability purposes)

### CODE REVIEW

File: packages/cw-orch-networks/src/networks/mod.rs [hunk 1]
```diff
@@ -31,46 +31,42 @@ pub use terra::{LOCAL_TERRA, PHOENIX_1, PISCO_1};
...
+    OSMO_5,
+    LOCAL_OSMO,
+    LOCAL_MIGALOO,
+    LOCAL_NEUTRON,
+    MIGALOO_1,
+    LOCAL_SEI,
+    SEI_DEVNET_3,
+    ATLANTIC_2,
+    PACIFIC_1,
+];
```
Comment (Buckram123) [inline packages/cw-orch-networks/src/networks/mod.rs:72]: Nice!

**Review by Buckram123 — APPROVED** [review]

File: packages/cw-orch-core/src/env.rs [hunk 2]
```diff
@@ -89,6 +91,16 @@ pub struct CwOrchEnvVars {
...
+
+    /// Optional - boolean
+    /// Defaults to "false"
+    /// Disable the "Enable Logs" message
+    /// It allows forcing cw-orch to not output anything
+    pub disable_logs_message: bool,
+    /// Optional - string
+    /// Specify a fee granter for interacting with a chain
+    /// This allows interacting with the blockchain and make another address pay for your fees (if allowed)
+    pub fee_granter: Option<String>,
```
[the remark this answers is not in the document]
Comment (Kayanski) [reply packages/cw-orch-core/src/env.rs:103]: Is it though

**Review by Buckram123 — APPROVED** [review]
Will be nice to have a little test with 3+ contracts ( could be in another PR )

File: cw-orch/src/snapshots.rs [hunk 3]
```diff
@@ -29,16 +29,17 @@ macro_rules! take_storage_snapshot {
         // We register and test a snapshot for all contracts storage
         use ::cw_orch::environment::{ChainState as _, StateInterface as _};
         let all_contract_addresses = $chain.state().get_all_addresses()?;
-        let mut all_storage = ::std::collections::HashMap::new();
+        let mut all_storage = vec![];
```
[the remark this answers is not in the document]
Comment (Kayanski) [reply cw-orch/src/snapshots.rs]: Hm, yeah better like so

**Pull request merged**